src/components/UI/Collapse/Collapse.tsx

```diff
@@ -1,46 +1,43 @@
+import React, { useState } from 'react'
+import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material'
 import css from './Collapse.module.css'
-import { useState } from 'react'
-import ReactMarkdown from 'react-markdown'
-import ExpandButton from '../ExpandButton/ExpandButton'
+import { ReactComponent as ChevronSvg } from '../../../assets/icons/chevron_green.svg'
+import { COLORS } from '../../../styles'
 
 export interface Props {
-	id: string
-	header: string
-	text: string
-	markdown?: boolean
+	header: React.ReactNode
+	children: React.ReactNode
+	expanded?: boolean
 }
 
 export function Collapse(props: Props): JSX.Element {
-	const close = { height: '0px' }
-	const [collapseStyle, setCollapseStyle] = useState(close)
+	const [opened, setOpened] = useState(props.expanded ?? false)
+	const accordionStyle: React.CSSProperties = {
```

Comment:
When customising a MUI component it is best practice to use the sx propety

Reply:
"MUi serves as a framework among many others, which are readily available for free, offering power and utility. However, our intention is to avoid coupling ourselves with any specific framework. Essentially, 'sx' performs the same as 'className,' providing us with two distinct methods to apply CSS styles. Consequently, new developers would need to familiarize themselves with both 'sx' and 'className.' In the future, another developer might introduce a new framework that uses 'gw, to set css'. This would result in three different methods for configuring CSS within the project. Having to learn three different approaches to achieve the same outcome, customizing CSS, could lead to confusion. 
Therefore, we must select a single approach that suits all scenarios, simplifying our code. Our goal is not to become overly dependent on any particular framework but to use them as we would any other component. Once we become tightly coupled in this manner, breaking away from it becomes exceedingly challenging."

Reply:
Understood, thank you for the explanation. 
Changing now